tests/testthat/test-entities_objects.R

```diff
@@ -0,0 +1,210 @@
+hashtags <- jsonlite::fromJSON('{
```

Comment:
I'd recommend putting the test data inside the tests to keep each test self contained.

Reply:
Moved them in

Reply:
It looks like you still define the data outside of `test_that()`?